superdesk/default_settings.py

```diff
@@ -616,3 +616,7 @@ def local_to_utc_hour(hour):
 # SAML Auth settings
 SAML_PATH = env('SAML_PATH')
 SAML_LABEL = env('SAML_LABEL', 'Single Sign On')
+
+EXCLUDED_VOCABULARY_FIELDS = env('EXCLUDED_VOCABULARY_FIELDS', 'default_categories,signal,rightsinfo,'
```

Comment:
imo this could be a list, I don't think we would ever change this via environment, it's easier to change it via settings file